tests/testthat/test-namespace.R

```diff
@@ -394,32 +394,63 @@ test_that("can extract non-imports from namespace preserving source", {
   expect_equal(namespace_exports(path), lines[c(1:3, 5)])
 })
 
-test_that("Invalid imports throw a helpful error", {
+test_that("invalid imports generate correct declarations", {
```

Comment:
@MichaelChirico I refactored into multiple tests since this one was getting a bit big.

Reply:
Looks good, thanks!